src/samplers/cpu/linux/usage/proc_stat.rs

```diff
@@ -12,8 +12,10 @@ pub struct ProcStat {
     interval: Interval,
     nanos_per_tick: u64,
     file: File,
-    counters_total: Vec<Counter>,
-    counters_percpu: Vec<Vec<DynBoxedMetric<metriken::Counter>>>,
+    total_counters: Vec<Counter>,
+    total_busy: Counter,
+    percpu_counters: Vec<Vec<DynBoxedMetric<metriken::Counter>>>,
```

Comment:
Afaict, this definition is the same as the `PercpuCounters` for the bpf code. Shall we hoist that structure outside to `src/common/mod.rs` to sit next to `Counter`, so both code paths can use it?